plugins/modules/azure_rm_webapp.py

```diff
@@ -100,7 +100,7 @@
         suboptions:
             name:
                 description:
-                    - Name of container, for example C(imagename:tag).
+                    - Name of the container, for example C(imagename:tag). To create a multi-container app, the name should be C(COMPOSE|) or C(KUBE|) followed by base64 encoded configuration.
```

Comment:
Why use such a naming method? Is this different from normal naming?
```suggestion
                    - Name of the container, for example C(imagename:tag).
                    - To create a multi-container app, the name should be 'COMPOSE|' or 'KUBE|' followed by base64 encoded configuration.
```

Reply:
This is the format that Azure seems to use to differentiate between single and multi-container applications. `az webapp config container show` returns the containers name in this format if webapp is a multi-container application.

Reply:
I see! Thank you very much!